Userland/Libraries/LibWebView/URL.cpp

```diff
@@ -35,7 +35,7 @@ static Optional<URL> create_url_with_url_or_path(String const& url_or_path)
 static Optional<URL> query_public_suffix_list(StringView url_string)
 {
     auto out = MUST(String::from_utf8(url_string));
-    if (!out.contains("://"sv))
+    if (!out.contains("about:"sv) && !out.contains("://"sv))
```

Comment:
Please add unit tests for this new case to Tests/LibWebView/TestWebViewURL.cpp.

Reply:
I think this should already be covered by the scheme check lower down. Like this it would match things like `https://example.com/about:`.

Reply:
Not covered by the case below unfortunately. Without a check here, we will form the string `https://about:blank", which is an invalid URL. So if you have search-via-the-URL-bar enabled in your settings, we will then search e.g. google for "about:blank".

Reply:
I do think @skyrising's point about matching things like "example.com/about:" is valid though - you can probably change this to `!out.starts_with("about:"sv)`? PR looks good to me otherwise.

Reply:
Sure I have fixed it